eclipse/ts.eclipse.ide.core/src/ts/eclipse/ide/internal/core/preferences/TypeScriptCorePreferenceInitializer.java

```diff
@@ -222,4 +218,50 @@ public static void fixEmbeddedPreference(IEclipsePreferences preferences) {
 			}
 		}
 	}
+
```

Comment:
Sorry for my too quick comments. Please re-read my propistion with IDETypeScriptRepositoryManager. TypeScriptRepositoryManager must not be linked to Eclipse (core folder could be consumed with other IDE)